snoop/views.py

```diff
@@ -162,29 +162,32 @@ def document_json(request, collection_slug, id):
 def feed(request, collection_slug):
     collection = get_object_or_404(models.Collection, slug=collection_slug)
 
-    ids = RawSQL("(SELECT id FROM snoop_document WHERE collection_id=%s)",
-        (collection.id,))
-    query = models.Digest.objects.filter(id__in=ids).order_by('-updated_at')
```

Comment:
This was all worth it to make the "copy each and every document id to python-managed memory and then push the list over to Django ORM" logic go away.

Job well done.